src/lib/application/files/js/src/main.js

```diff
@@ -4,6 +4,8 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
```

Comment:
is this one extra line really required? I would suggest staying consistent with the docs

Reply:
Updated.